impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -143,5 +143,9 @@ std::shared_ptr<DeviceBuffer> AllocatorVK::CreateBuffer(StorageMode mode,
   return std::make_shared<DeviceBufferVK>(length, mode, context_,
                                           std::move(device_allocation));
 }
-
+ISize AllocatorVK::GetMaxTextureSizeSupported() {
+  // source :
+  // https://registry.khronos.org/vulkan/specs/1.2-extensions/html/vkspec.html#limits-minmax
+  return {4096, 4096};
```

Comment:
I believe these are just required limits. The actual supported may be much larger. Let's add a TODO with a bug to figure this out for Vulkan later.

Reply:
Done.